Tutorials/whatiswot/website/docs/wot/application-domains/industrial-and-agriculture-systems.md

```diff
@@ -0,0 +1,53 @@
+---
+sidebar_label: Industrial and Agriculture Systems
+---
+
+# Industrial and Agriculture Systems
+
+![coming-soon](/img/tutorial/What-Is-Wot/coming_soon_banner.png)
+
+The Web of Things finds applications in diverse fields beyond smart homes and cities. This tutorial will focus on industrial and agriculture systems - two major and complex examples.
```

Comment:
also adding that we will show how all the domains can collaborate with each other thanks to WoT